DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -44,8 +45,8 @@
       <div class="nhsuk-width-container nhsuk-header__container">
         <div class="nhsuk-header__logo">
           <a class="nhsuk-header__link nhsuk-header__link--service"
-             href="@(ViewData["HeaderPath"] ?? @Configuration["CurrentSystemBaseUrl"] + "/Home?action=appselect")"
-             aria-label="NHS Digital Learning Solutions - @(ViewData["HeaderPathName"] ?? "Switch Application")">
+             href="@(ViewData["HeaderPath"] ?? @Configuration["CurrentSystemBaseUrl"] + "/Home")"

```

Comment:
The Configuration["CurrentSystemBaseUrl"] is used to point to the old site. Configuration["AppRootPath"] points to the new site, and I think that should be used instead for most of these links. In theory, these should end up the same, but in the dev environment, we probably want AppRootPath so we can navigate between the different links without running the old code. The ticket mentions which links "currently navigates to old site" but I'm not sure if that means the new links should point there as well. I reckon the Welcome link should use AppRootPath but FindYourCentre should use CurrentSystemBaseUrl, but I think we'd need some clarity on that @stellake

Reply:
"Currently navigates to the old site" means that we should navigate to old site, but this will be changed in the future - sorry about the slightly confusing wording! 🙏 

I think doing something special in dev is not necessary - I'd just use AppRootPath for new pages & CurrentSystemBaseUrl for old pages. :)